webapp-django/crashstats/base/jinja2/footer_nav.html

```diff
@@ -3,10 +3,8 @@
         <b>Mozilla Crash Reports</b> - Powered by <a href="https://github.com/mozilla-services/socorro">Socorro</a> - All dates are UTC
     </div>
     <ul>
-        <li><a href="{{ url('documentation:home') }}">User Documentation</a></li>
-        <li><a href="{{ url('api:documentation') }}">API</a></li>
```

Comment:
All docs are being pulled into a single documentation area including the API docs.